corehq/apps/telerivet/tasks.py

```diff
@@ -16,6 +17,28 @@
 
 @task(queue=CELERY_QUEUE)
 def process_incoming_message(*args, **kwargs):
+    try:
+        log = IncomingRequest(
+            event=kwargs["event"],
+            message_id=kwargs["message_id"],
+            message_type=kwargs["message_type"],
+            content=kwargs["content"],
+            from_number=kwargs["from_number"],
+            from_number_e164=kwargs["from_number_e164"],
+            to_number=kwargs["to_number"],
+            time_created=kwargs["time_created"],
+            time_sent=kwargs["time_sent"],
+            contact_id=kwargs["contact_id"],
+            phone_id=kwargs["phone_id"],
+            service_id=kwargs["service_id"],
+            project_id=kwargs["project_id"],
+            secret=kwargs["secret"],
```

Comment:
maybe there is extra stuff in there? seems like you could clean this up a little given how repetitive this is.